test/specs/ngeo/modules/import/WmsGetCapDirective.spec.js

```diff
@@ -1,101 +0,0 @@
-/* eslint-disable max-len */
-describe('ngeo.wmsGetCapDirective', function() {
-  var elt, scope, parentScope, map, $rootScope, $compile, $window;
-
-  var loadDirective = function() {
-    parentScope = $rootScope.$new();
-    var tpl = '<div ngeo-wms-get-cap="getCap" ngeo-wms-get-cap-map="map" ngeo-wms-get-cap-options="options"></div>';
-    elt = $compile(tpl)(parentScope);
-    $rootScope.$digest();
-    scope = elt.isolateScope();
-  };
-
-  var wmsBasic;
-
-  before(function(done) {
-    $.get('base/test/data/wms-basic.xml', function(response) {
-      wmsBasic = response;
-      done();
-    });
-  });
-
-  beforeEach(function() {
-
-    module(function($provide) {
-      $provide.value('gaLayers', {});
-      $provide.value('gaTopic', {});
-      $provide.value('gaLang', {
-        get: function() {
-          return 'somelang';
-        },
-        getNoRm: function() {
-          return 'somelang';
-        }
-      });
-    });
-
-    inject(function($injector) {
-      $rootScope = $injector.get('$rootScope');
-      $compile = $injector.get('$compile');
-      $window = $injector.get('$window');
-    });
-
-    map = new ol.Map({});
-    map.setSize([600, 300]);
-  });
-
-  describe('loads a correct WMS GetCapabilities', function() {
```

Comment:
I think this one is missing in ga tests

Reply:
no it exist in https://github.com/geoadmin/mf-geoadmin3/blob/5a0534eeb919d89d8c7fd54a8fd04a752f32f6cd/test/specs/import/WmsGetCapDirective.spec.js#L47